util/block.go

```diff
@@ -38,7 +39,7 @@ type Block struct {
 	blockHash       *daghash.Hash  // Cached block hash
 	transactions    []*Tx          // Transactions
 	txnsGenerated   bool           // ALL wrapped transactions generated
-	blueScore       uint64         // Blue score
+	blueScore       *uint64        // Blue score
```

Comment:
Please, unless you already done that, make sure nobody is using blueScore directly.
Add to comment about Blue score that it shouldn't be used, and use .BlueScore instead